extensions/docs/source/index.rst

```diff
@@ -13,7 +13,7 @@ Compile time Extension Points for Hyperledger Fabric by leveraging `Go Modules <
    extension
    Contribution <https://github.com/trustbloc/community/blob/master/CONTRIBUTING.md>
    questions
-   hyperledgerfabric
+   Hyperledger Fabric <https://hyperledger-fabric.readthedocs.io/en/release-1.4/>
```

Comment:
Should link to v2 docs rather than 1.4

Reply:
Can you confirm if its this one https://hyperledger-fabric.readthedocs.io/en/latest/
As its says latest only no v2